src/Cli/dotnet/Commands/Run/RunProperties.cs

```diff
@@ -1,14 +1,12 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#nullable disable warnings
-
 using Microsoft.Build.Execution;
 using Microsoft.DotNet.Cli.Utils;
 
 namespace Microsoft.DotNet.Cli.Commands.Run;
 
-internal record RunProperties(string? RunCommand, string? RunArguments, string? RunWorkingDirectory)
+internal record RunProperties(string RunCommand, string? RunArguments, string? RunWorkingDirectory)
```

Comment:
This is another place where having arguments, specifically `RunArguments`, be potentially `null` feels potentially off and `""` may work.